### PR TITLE
Prepare applications for use with Launchpad Service and HTML5 repo

### PR DESCRIPTION
**Pull request by uwe-klinger**
The changes are a precondition to deploy the UI5 applications in the `app` folder to the HTML5 repo and use them in the SAP BTP Launchpad Service. The changes for the `mta.yaml` deployment descriptor are not included.

* Added `package.json` required to run the UI5 build
* Added `applicationVersion` in `manifest.json` files to deploy to HTML5 repo
* Added `crossNavigation` in `manifest.json` files to add application as Launchpad tiles

### CODE REVIEW

File: app/admin/webapp/manifest.json [hunk 1]
```diff
@@ -18,6 +21,18 @@
             "id": "ui5template.basicSAPUI5ApplicationProject",
             "-id": "ui5template.smartTemplate",
             "-version": "1.40.12"
+        },
+        "crossNavigation": {
+            "inbounds": {
+                "Admin-manage": {
```
Comment (beckermarc) [inline app/admin/webapp/manifest.json:27]: Rather `Books-manage`

File: app/addresses/webapp/manifest.json [hunk 2]
```diff
@@ -18,6 +21,18 @@
             "id": "ui5template.basicSAPUI5ApplicationProject",
             "-id": "ui5template.smartTemplate",
             "-version": "1.40.12"
+        },
+        "crossNavigation": {
+            "inbounds": {
+                "Addresses-manage": {
```
Comment (beckermarc) [inline app/addresses/webapp/manifest.json:27]: Rather `Addresses-browse` or ``Addresses-show`

File: app/browse/webapp/manifest.json [hunk 3]
```diff
@@ -18,6 +21,18 @@
             "id": "ui5template.basicSAPUI5ApplicationProject",
             "-id": "ui5template.smartTemplate",
             "-version": "1.40.12"
+        },
+        "crossNavigation": {
+            "inbounds": {
+                "Books-manage": {
```
Comment (beckermarc) [inline app/browse/webapp/manifest.json:27]: Rather `Books-browse` or `Books-show`

File: app/admin/webapp/manifest.json (hunk 1 above)
Comment (uwe-klinger) [reply app/admin/webapp/manifest.json]: changed

File: app/addresses/webapp/manifest.json (hunk 2 above)
Comment (uwe-klinger) [reply app/addresses/webapp/manifest.json]: changed

File: app/browse/webapp/manifest.json (hunk 3 above)
Comment (uwe-klinger) [reply app/browse/webapp/manifest.json]: changed

**Review by beckermarc — APPROVED** [review]

**Pull request merged**